.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   },
   "extends": "airbnb",
   "rules": {
+    "linebreak-style": 0,
```

Comment:
Yes you're right.
I'll change it on all project and configure my IDE accordingly